public/docs/ts/latest/glossary.jade

```diff
@@ -54,7 +54,7 @@ include _util-fns
     The barrel itself is a module file that re-exports *selected* exports of other modules.
     
     Imagine three modules in a `heroes` folder:
-  code-example(format='').
+  code-example.
```

Comment:
Short answer: because it has no effect `code-example` (see the [source](https://github.com/angular/angular.io/blob/master/public/resources/js/directives/code-example.js)).

Longer answer: this idiom comes from the `makeExample` mixin (where it prevents the default behavior of turning on line numbers if there is more than one line). But is has no utility when applied directly to `code-example`.